docs/pages/hooks/useIdentityKit.mdx

```diff
@@ -7,7 +7,13 @@ authors:
 
 # useIdentityKit
 
-Hook for getting the agent for the currently connected signer.
+Hook for getting currently connected user, agent, his ICP balance and disconnect:
```

Comment:
Hooks for getting the agent for the currently connected user, its wallet address (as a valid ICRC1 Account), balance, and disconnect function:

Reply:
done